### PR TITLE
TECH-12478 - Call Ingestion Documentation Updates (version 2022-08-01)

### PR DESCRIPTION
**Pull request by kamerynn**
[TECH-12478 - Consolidate and improve documentation for Call Ingestion onboarding](https://invoca.atlassian.net/browse/TECH-12478)

Updates Call Ingestion API documentation with:
- A focus on requirements, such as stereo channel requirement
- A mention of our 24 hour SLO
- Add recording access option describing the recording proxy (Direct Recording Access)
- Separate the Example requests out, which in turn makes them easier to see on documentation
- Some opinionated and debatable changes on formatting, order of sections, etc.

A barrage of screenshots from my local build below:

Screenshot 2023-08-28 at 4 32 10 PM

Screenshot 2023-08-28 at 4 33 37 PM

Screenshot 2023-08-28 at 4 33 42 PM

Screenshot 2023-08-28 at 4 33 49 PM

Screenshot 2023-08-28 at 4 34 05 PM

## Checklist

- [ ] Find the [Service owning team](https://docs.google.com/spreadsheets/d/1YF2wuepY5SZTpVEdT9gwhTPhpnDGZeVjywEwvbJS5TQ/edit#gid=0&fvid=1520238175) for these changes, and tag the team as "Reviewers" on this PR
- [ ] Test the documentation changes on readthedocs as a private branch
- [] If changing general content, have agreement on whether to apply to latest version or all versions (if all versions, provide links to the related PRs below)

### CODE REVIEW

**Comment by kamerynn** [comment]
@ryanbales @sasha-romanova I don't have a private version of the docs yet, as The [build is failing](https://readthedocs.com/projects/invoca-developer-docs/builds/1678168/) on a new requirement for a configuration file which doesn't exist. I'll figure that out and commit it in this PR. In the meantime, though, I added a bunch of screenshots for the changes. I'm hoping that's sufficient for a first-pass review of the documentation. Thank you!

**Comment by kamerynn** [comment]
Also cc @invoca-ben-hafer if you were wondering why your recent doc changes aren't showing up on `latest`
> The [build is failing](https://readthedocs.com/projects/invoca-developer-docs/builds/1678168/) on a new requirement for a configuration file which doesn't exist. I'll figure that out and commit it in this PR

File: source/api_documentation/call_ingestion_api/index.rst
Comment (sasha-romanova) [inline source/api_documentation/call_ingestion_api/index.rst]: the name of the param is actually `recording_filename` - easy thing to miss. I think in other references you have it right.

**Review by sasha-romanova — APPROVED** [review]
LGTM apart from changing 'filepath' to 'filename'

File: source/api_documentation/call_ingestion_api/index.rst
Comment (kamerynn) [reply source/api_documentation/call_ingestion_api/index.rst]: oof. thank you. I'll get this fixed

**Comment by kamerynn** [comment]
The build issues were resolved in this hotfix: https://github.com/Invoca/developer-docs/pull/329 (cc @invoca-ben-hafer your changes should be available on documentation now)

I'm going to consider this PR good to go - once we merge to 2022-08-01 it should work fine.

However, I'm going to also copy these changes over to 2022-03-01, since thats the actual API version for Call Ingestion and these changes still apply to that version. I'll send both to get merged at the same time. I may have to propagate the hotfix into the other main branches..

**Comment by kamerynn** [comment]
@Invoca/integrations could I get a merge?

**Review by emakin1 — APPROVED** [review]

**Review by ryanbales — APPROVED** [review]
LGTM

**Pull request merged**
